framework/src/modules/dynamic_reward/utils.ts

```diff
@@ -23,15 +23,16 @@ export const getMinimalRewardActiveValidators = (
 	(defaultReward * BigInt(moduleConfig.factorMinimumRewardActiveValidators)) /
 	DECIMAL_PERCENT_FACTOR;
 
-export const getStakeRewardActiveValidators = (
+export const getStakeRewardActiveValidators = async (
```

Comment:
I think you could avoid using context and validatorMethod (therefor no need to import ValidatorsMethod) and just use `numberActiveValidators` as an argument here. This would also be faster since there is no need to call the validators module here at all. 

Is there any other reason that I am missing to do it like this here?